enlace/programs/profile.php

```diff
@@ -852,18 +851,27 @@ function(response) {
                             $all_participants = mysqli_query($cnnEnlace, $get_all_participants);
                             while ($all_p = mysqli_fetch_array($all_participants)) {
-                                $find_absence = "SELECT COUNT(*) FROM Absences WHERE Participant_ID='" . $all_p['Participant_ID'] . "'
+                                $find_absence = "SELECT Absent FROM Absences WHERE Participant_ID='" . $all_p['Participant_ID'] . "'
                         AND Program_Date='" . $temp_program['Program_Date_ID'] . "'";
                                 $absent = mysqli_query($cnnEnlace, $find_absence);
                                 $num_abs = mysqli_fetch_row($absent);
-                                $was_absent = $num_abs[0];
+                                $was_there = false;
+                                $was_absent = false;
+                                if($num_abs) {
+                                    $was_there = $num_abs[0] == 0;
+                                    $was_absent = $num_abs[0] == 1;
+                                }
 
                                 /* uncheck box to add an absence, and check the box to delete that absence: */
-                                    ?><input type="checkbox" <?php echo($was_absent <= 0 ? 'checked="true"' : null); ?>
-                                               id="date_<?php echo $temp_program['Program_Date_ID']; ?>_person_<?php echo $all_p['Participant_ID'] ?>"
-                                               onchange="handleRole(this, '<?php echo $temp_program['Program_Date_ID']; ?>', '<?php echo $all_p['Participant_ID'] ?>');">
-            <?php echo $all_p['First_Name'] . " " . $all_p['Last_Name'];
-            ?><br><?php
+                                    ?>
+                                <span style="font-weight:bold"><?php echo $all_p['First_Name'] . " " . $all_p['Last_Name']; ?></span>
+                                Attended: <input type="checkbox" <?php echo($was_there ? 'checked="true"' : null); ?>
+                                               id="present_date_<?php echo $temp_program['Program_Date_ID']; ?>_person_<?php echo $all_p['Participant_ID'] ?>"
+                                               onchange="markPresent(this, '<?php echo $temp_program['Program_Date_ID']; ?>', '<?php echo $all_p['Participant_ID'] ?>');">
+                                Absent: <input type="checkbox" <?php echo($was_absent ? 'checked="true"' : null); ?>
+                                               id="absent_date_<?php echo $temp_program['Program_Date_ID']; ?>_person_<?php echo $all_p['Participant_ID'] ?>"
+                                               onchange="markAbsent(this, '<?php echo $temp_program['Program_Date_ID']; ?>', '<?php echo $all_p['Participant_ID'] ?>');">
+                                               <br><?php

```

Comment:
Some style notes here in a separate comment, but also the labels should use the html `<label>` tag, so that clicking them activates the checkboxes appropriately (and for accessibility).